testsuite/podman_runner/18_run_secondary_tests.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+
 set -xe
 TESTS="$@"
-sudo -i podman exec controller bash -c "export SCC_CREDENTIALS=\"test|test\" && export GITPROFILES=https://github.com/uyuni-project/uyuni.git#:testsuite/features/profiles/internal_nue && export PXEBOOT_IMAGE=sles15sp4 && export BUILD_HOST=buildhost && export AUTH_REGISTRY=${AUTH_REGISTRY} && export AUTH_REGISTRY_CREDENTIALS=\"${AUTH_REGISTRY_CREDENTIALS}\" && export NO_AUTH_REGISTRY=${NO_AUTH_REGISTRY} && export CUCUMBER_PUBLISH_TOKEN=${CUCUMBER_PUBLISH_TOKEN} && export PROVIDER=podman && export SERVER=server && export HOSTNAME=controller && export SSH_MINION=opensusessh && export MINION=sle_minion && export RHLIKE_MINION=rhlike_minion && export TAGS=\"\\\"not @flaky\\\"\" && export DEBLIKE_MINION=deblike_minion && cd /testsuite && rake cucumber:secondary ${TESTS}"
+sudo -i podman exec controller bash -c "export SCC_CREDENTIALS=\"test|test\" && export GITPROFILES=https://github.com/jordimassaguerpla/uyuni.git#ci_fix_docker_auth_registry:testsuite/features/profiles/github_runner && export PXEBOOT_IMAGE=sles15sp4 && export BUILD_HOST=buildhost && export AUTH_REGISTRY=${AUTH_REGISTRY} && export AUTH_REGISTRY_CREDENTIALS=\"${AUTH_REGISTRY_CREDENTIALS}\" && export NO_AUTH_REGISTRY=${NO_AUTH_REGISTRY} && export CUCUMBER_PUBLISH_TOKEN=${CUCUMBER_PUBLISH_TOKEN} && export PROVIDER=podman && export SERVER=server && export HOSTNAME=controller && export SSH_MINION=opensusessh && export MINION=sle_minion && export RHLIKE_MINION=rhlike_minion && export TAGS=\"\\\"not @flaky\\\"\" && export DEBLIKE_MINION=deblike_minion && cd /testsuite && rake cucumber:secondary ${TESTS}"
```

Comment:
we need to fix the profile in master upstream and here use https://github.com/uyuni-project/uyuni.git#:testsuite/features/profiles/github_runner

Reply:
once this is merged https://github.com/uyuni-project/uyuni/pull/10061